gym/lib/gym/options.rb

```diff
@@ -200,15 +166,6 @@ def self.plain_options
                                      description: "The toolchain that should be used for building the application (e.g. com.apple.dt.toolchain.Swift_2_3, org.swift.30p620160816a)",
                                      optional: true,
                                      is_string: false),
-        FastlaneCore::ConfigItem.new(key: :provisioning_profile_path,
```

Comment:
Yeah it will fail, _fastlane_ will tell the user that this option isn't available.